src/bin/wc.rs

```diff
@@ -103,6 +103,10 @@ struct Flags {
     count_bytes: bool,
 }
 
+fn u64_num_digits(val: u64) -> usize {
+    (val as f32).log10() as usize + 1
```

Comment:
All I can say is that it is wrong. It might even [cause UB](https://github.com/rust-lang/rust/issues/32242), when given 0. Furthermore `log` on floats is slow (but, tbh, that doesn't matter), it will be an order of magnitude faster to just have a recursive algorithm, dividing by ten.